index.html

```diff
@@ -2,6 +2,9 @@
 <html lang="en">
 
 <head>
+   <style>

```

Comment:
It's not a good practice to add styling to the head tag. There is a separate [css](https://github.com/Evavic44/Hotel-Zeitaku/blob/main/css/style.css) file for that.